client/data/marketplace/constants.ts

```diff
@@ -1,9 +1,101 @@
 export const BASE_STALE_TIME = 1000 * 60 * 60 * 2; // 2 hours
-export const DEFAULT_PAGE_SIZE = 24;
-export const DEFAULT_FIRST_PAGE = 1;
+export const DEFAULT_PAGE_SIZE = 20;
+export const DEFAULT_FIRST_PAGE = 0;
 export const DEFAULT_CATEGORY = 'all';
 
 export const SITE_SEARCH_CACHE_KEY = 'wpcom-site-search-wporg-plugins';
 export const WPORG_PLUGINS_BLOG_ID = '108986944';
+export const WCCOM_PLUGINS_BLOG_ID = '113771570';
 
 export const WPORG_CACHE_KEY = 'wporg-plugins';
+
+export const RETURNABLE_FIELDS = [
+	'author',
+	'author_login',
+	'blog_name',
+	'blog_icon_url',
+	'blog_id',
+	'category.name.default',
+	'comment_count',
+	'content..*.word_count',
+	'date',
+	'date_gmt',
+	'excerpt.default',
+	'excerpt_html',
+	'gravatar_url',
+	'has..*',
+	'image.url.raw',
+	'image.alt_text',
+	'like_count',
+	'modified',
+	'modified_gmt',
+	'permalink.url.raw',
+	'post_id',
+	'post_type',
+	'shortcode_types',
+	'tag.name.default',
+	'title.default',
+	'title_html',
```

Comment:
Do we need `title_html`?

Reply:
I have included all the possible fields currently to check the response but I will reduce that to the very minimum once we have clear what fields are needed. Meanwhile I will add a code comment indicating that.